src/transformState.js

```diff
@@ -5,7 +5,35 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  let object;
+
+  for (let i = 0; i < actions.length; i++) {
```

Comment:
Lets improve your knowledge and practice with another loop. Try [`for ... of`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...of)